biofefi/machine_learning/learner.py

```diff
@@ -433,6 +455,62 @@ def _evaluate(
     return eval_res
 
 
+def calculate_regression_metrics(y_true: np.array, y_pred: np.array, metric_function):
+    """
+    Calculate regression metrics for a given model.
+
+    Args:
+        - y_true (np.ndarray): True labels.
+        - y_pred (np.ndarray): Predicted labels.
+        - metric: Metric to calculate.
+
+    Returns:
+        - float: Value of the metric.
+    """
+    metric = metric_function(y_true, y_pred)
+
+    return metric
+
+
+def calculate_classification_metrics(
```

Comment:
If this function is only used in this module, you can add an underscore to the front of the name to indicate it's private. That's the convention in python.